apps/workers/src/main.ts

```diff
@@ -16,7 +16,6 @@ import {
     workerErrorHandlerService,
 } from './app/lib/di'
 import env from './env'
-import { cleanUpOutdatedJobs } from './utils'
```

Comment:
add back, these all got removed when resolving merge conflicts I'm imagining

Reply:
Strange, git did everything automatically without asking me for confirmation for conflicts in the merge but I think this is certainly the reason.